x-pack/legacy/plugins/maps/server/maps_telemetry/maps_telemetry.ts

```diff
@@ -184,8 +184,9 @@ export async function getMapsTelemetry(
     showMapVisualizationTypes: config().get('xpack.maps.showMapVisualizationTypes'),
   };
   const mapsTelemetry = buildMapsTelemetry({ mapSavedObjects, indexPatternSavedObjects, settings });
-  return await savedObjectsClient.create(TELEMETRY_TYPE, mapsTelemetry, {
+  const { attributes } = await savedObjectsClient.create(TELEMETRY_TYPE, mapsTelemetry, {
```

Comment:
So this bug was this due to a change in how `SavedObjectClient#create` returned its results?

Reply:
so `attributes` and `mapsTelemetry` are identical then?

Reply:
> So this bug was this due to a change in how SavedObjectClient#create returned its results?

Correct. Or at least how we're using those results. I think previously we were getting the `attributes` on the calling side.

> so attributes and mapsTelemetry are identical then?

This was bothering me too when I made these updates. From what I can tell, we're not using this saved object. I think it's a relic of when we were using telemetry in conjunction with task manager. We removed the task manager logic a while back but this appears to have been left behind. I've removed it which makes things a bit simpler. Ran the local tests and checked the stats endpoint and don't see any adverse effects!